README.md

```diff
@@ -45,17 +45,16 @@ The user can either `sign up`, `sign in` or `sign up or in`
 ```go
 import (
     "github.com/descope/go-sdk/descope"
-    "github.com/descope/go-sdk/descope/auth"
 )
 
 // Every user must have a loginID. All other user information is optional
 loginID := "desmond@descope.com"
-user :=  &auth.User{
+user := &descope.User{
```

Comment:
do we need to also add `import` statement for the `descope` package in the example? I think we should